Makefile

```
@@ -42,16 +44,39 @@ test:
 format:
 	@echo "Formatting code..."
 	pnpm format
 
 # --- Deploy ---
 
-deploy: deploy-api deploy-bot deploy-worker
+# Run quality gates serially before deploying so `make -j deploy` cannot
+# start a deploy before checks finish. Mirror the CI deployment order:
+# build cf-shared, generate version files, apply D1 migrations, then deploy
+# each service so shared code and schema are never stale.
+deploy:
```

checkmake (0.3.2)

[warning] Target body for "deploy" exceeds allowed length of 5 lines (6).

(maxbodylength)